library/std/src/sys_common/mod.rs

```diff
@@ -44,7 +44,7 @@ cfg_if::cfg_if! {
 
 cfg_if::cfg_if! {
     if #[cfg(any(
-        all(unix, not(target_os = "l4re")),
+        all(unix, not(target_os = "l4re"), not(target_os = "zkvm")),
```

Comment:
The target doesn't set `target-family: unix`, so this not-clause should be unnecessary.

Reply:
Thanks for catching this!